src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -21,7 +21,18 @@
             transportSend(type, args);
         };
     });
-    
+
+    // Implements error handling
+    window.addEventListener("error", function(messageOrEvents) {
+        var args = {
+            "message": messageOrEvents["message"],
+            "line": messageOrEvents["error"]["lineNumber"],
+            "stack": messageOrEvents["error"]["stack"]
```

Comment:
NOTE: in an object literal (vs. JSON), when dealing with keys as `String`, you don't need to quote.  Also, I think we should be careful not to crash if some browser doesn't provide the correct things:

```js
var message = messageOrEvents.message;
var error = messageOrEvents.error || {};
var args = {
    message: message,
    line: error.lineNumber,
    stack: error.stack
};
```